lib/sass_spec/test.rb

```diff
@@ -1,9 +1,16 @@
 require 'minitest'
 
+parallelize = false
+
 def run_spec_test(test_case, options = {})
   if options[:skip_todo] && test_case.todo?
     skip "Skipped todo"
   end
+  # When running executables we can safely parallelize
+  # Executables must become their own process environment
+  if options[:engine_adapter].instance_of? (ExecutableEngineAdapater)
```

Comment:
Will remove the commit again, had a thinking error. It would only work if we redirect stderr after the process is started. This still has a race condition between redirection of stderr and open3.